src/components/popover-menu/popover-menu.tsx

```diff
@@ -0,0 +1,37 @@
+import { h, Component } from "preact";
```

Comment:
@semarche-kaltura 
I really like this component. 
Can you add a readme in the root of this component elaborating (not too long) on how to use this with like examples of simple use and with custom component? 
At some point we will have a storybook that will do this for us, but this would be a cool to get a bit of dodumentation for a component you built.

Reply:
done